0000-template.md

```diff
@@ -1,7 +1,11 @@
 
-- Start Date: (fill me in with today's date, YYYY-MM-DD)
-- RFC PR: (leave this empty)
-- FOLIO Issue: (leave this empty)
+- Stage: (DRAFT REVIEW, PRELIMINARY REVIEW, PUBLIC REVIEW, FINAL REVIEW)
+- RFC PRs: 
+  - DRAFT REVIEW: TBD
+  - PRELIMINARY REVIEW: TBD
+  - PUBLIC REVIEW: TBD
+  - FINAL REVIEW: TBD
+- Outcome:  (Leave this blank.  Will eventually be either ACCEPTED or REJECTED)
```

Comment:
would it make sense if a outcome of REJECTED but the file would still be in this repo?

Thinking that the presence of the file in this repo means that it is accepted. A decent branching strategy could allow this to happen.

Reply:
IMO RFCs, even those which have been rejected, should remain in the repo.  Otherwise, the content and conversations are much more difficult to find.  We don't want to lose that information.

Reply:
I am good with this.